src/oscar/views/generic.py

```diff
@@ -57,8 +57,8 @@ def get_checkbox_object_name(self):
     def get_error_url(self, request):
         return safe_referrer(request, '.')
 
-    def get_success_url(self, request):
-        return safe_referrer(request, '.')
+    # def get_success_url(self, request):
```

Comment:
I still don't see how this method has anything to do with this PR? Can't you just keep it like it was?

Reply:
There was a codecov error concerning this function, so I commented it out to have all checks passed. It's not needed any more by ranges.views.py